runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -4702,6 +4714,23 @@ void memoryDisclaimLogic(TR::CompilationInfo *compInfo, uint64_t crtElapsedTime,
 #endif // J9VM_INTERP_PROFILING_BYTECODES
    }
 
+void rssReportLogic(TR::CompilationInfo *compInfo)
```

Comment:
Who is calling this new function?

Reply:
I added this comment:
`// this method can be used to produce regular RSS reports`